### PR TITLE
API-1562 Latest Resource ETL Status Entity Creation

### PR DESCRIPTION
**Pull request by that-guy-is-it**

### CODE REVIEW

File: data-query/src/main/java/gov/va/api/health/dataquery/service/controller/latestresourceetlstatus/LatestResourceEtlStatusEntity.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import org.springframework.data.domain.Sort;
+
+@Data
+@Entity
+@Builder
+@Table(name = "Latest_Resource_ETL_Status", schema = "app")
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+public class LatestResourceEtlStatusEntity extends Object {
```
Comment (jblefkove-va) [inline data-query/src/main/java/gov/va/api/health/dataquery/service/controller/latestresourceetlstatus/LatestResourceEtlStatusEntity.java:23]: `extends Object` is redundant... in Java all classes extend `Object` either directly or indirectly

File: data-query/src/main/java/gov/va/api/health/dataquery/service/controller/latestresourceetlstatus/LatestResourceEtlStatusEntity.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+  @Id
+  @Column(name = "ResourceName")
+  @EqualsAndHashCode.Include
+  private String resourceName;
+
+  @Column(name = "EndDateTimeUTC")
+  private Instant endDateTime;
+
+  static Sort naturalOrder() {
```
Comment (jblefkove-va) [inline data-query/src/main/java/gov/va/api/health/dataquery/service/controller/latestresourceetlstatus/LatestResourceEtlStatusEntity.java:33]: Don't include this unless it's being used somewhere

File: data-query/src/main/java/gov/va/api/health/dataquery/service/controller/latestresourceetlstatus/LatestResourceEtlStatusEntity.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
+package gov.va.api.health.dataquery.service.controller.latestresourceetlstatus;
```
Comment (jblefkove-va) [inline data-query/src/main/java/gov/va/api/health/dataquery/service/controller/latestresourceetlstatus/LatestResourceEtlStatusEntity.java:1]: Package name is too specific. @bschofield-va is `gov.va.api.health.dataquery.service.controller` sufficient or should there be a dedicated healthcheck package?

Comment (bschofield-va) [reply data-query/src/main/java/gov/va/api/health/dataquery/service/controller/latestresourceetlstatus/LatestResourceEtlStatusEntity.java:1]: I think a dedicate package for this. Perhaps, `...controller.etlstatus` ?

**Review by jblefkove-va — APPROVED** [review]

**Pull request merged**